src/App.tsx

```diff
@@ -1,8 +1,98 @@
 import React from 'react';
-import './App.css';
+import { observer } from 'mobx-react-lite';
+import { Heading, Text, Box } from 'rebass';
+import partition from 'lodash/partition';
+import { store } from './store';
+import { PendingBets } from './PendingBets';
```

Comment:
Proposals:
- use default exports, since they are the only components we export from files.
- move those 3 into components folder and import all at once
```
import { PendingBets, CompletedBets, PrimeForm } from './components'
```

Reply:
I'm not a big fan of default exports in ts code. For example, named exports much better for auto-imports, renaming (in VS Code you can rename export and it will be renamed everywhere)

Reply:
Dismiss this one then :)